cea/datamanagement/format_helper/cea4_migrate.py

```
@@ -0,0 +1,411 @@
+
+
+__author__ = "Zhongming Shi"
+__copyright__ = "Copyright 2025, Architecture and Building Systems - ETH Zurich"
+__credits__ = ["Zhongming Shi"]
+__license__ = "MIT"
+__version__ = "0.1"
+__maintainer__ = "Reynold Mok"
+__email__ = "cea@arch.ethz.ch"
+__status__ = "Production"
+
+from cea.datamanagement.format_helper.cea4_verify import cea4_verify, verify_shp, CSV_BUILDING_PROPERTIES_4, \
```

Ruff (0.8.2)

`cea.datamanagement.format_helper.cea4_verify.CSV_BUILDING_PROPERTIES_4` imported but unused

Remove unused import: `cea.datamanagement.format_helper.cea4_verify.CSV_BUILDING_PROPERTIES_4`

(F401)